tools/devops/automation/templates/mac/build.yml

```diff
@@ -147,7 +147,7 @@ steps:
 
     [System.Collections.Generic.List[string]]$failures = @()
 
-    $macTest = @("dontlink", "apitest", "introspection", "linksdk", "linkall", "xammac_tests")
+    $macTest = @("dontlink", "introspection", "linksdk", "linkall", "xammac_tests")
```

Comment:
but `monotouch-test` was added for `Mac [dotnet]`
and we should start to run tests on Catalyst (in 10.15+)

Reply:
This are only ran Catalina and later, should we add those? We might just want to target catalina, meaning is a much bigger change in the pipeline definition.

Reply:
> https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md

10.14+ so Mojave should run `Mac [dotnet]` too... ideally

Reply:
We can try #YOLO